setup_scripts/setup_realsense.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
It would be helpful to have a quick comment here explaining what this file is for (it's association with the realsense package, why we need it, etc.)

Reply:
Added